### DISCUSSION TITLE
Can't find how to change the sidebar tabs.

### DISCUSSION THREAD

**Discussion by Lightmarey** [soulhotel/FF-ULTIMA#201] [Q&A] [answered]
I use multiple sidebar tools like sideberry and tabstash.
In previous versions, there was a selection button on the sidebar for switching. However, in version 1.9.5, I can no longer switch sidebar tools quickly. 
I've looked at all the options provided by ultima and searched about "sidebar" in about:config, and still can't find a solution. Can anyone please help me.
<img width="972" alt="EGlY8jxVhR" src="https://github.com/user-attachments/assets/01c582c8-5273-4e08-ac22-e30b26c19206">

**Comment by soulhotel** [comment, marked as answer]
Firefox removed the Sidebar Header when they implemented vertical tabs. It doesn't exist anymore.

See the [Sidebery only Wiki](https://github.com/soulhotel/FF-ULTIMA/wiki/Sidebery-Configuration), its not the best approach, but its a solution.

**Reply by Lightmarey** [reply]
Thank you a lot.

**Reply by soulhotel** [reply]
Np, also experiment with turning `nightly.override` on/off when using the indicator setting, depending on what firefox edition you are on, it may help a bit more.
